src/lib/components/atoms/title/title.constants.ts

```diff
@@ -10,8 +10,8 @@ export const HEADING_TO_SIZE: Record<TitleTag, TitleSize> = {
 }
 
 export const SIZE_TO_HEADING: Record<TitleSize, TitleTag> = {
-    [TitleSize.Large]: TitleTag.H1,
-    [TitleSize.Medium]: TitleTag.H2,
-    [TitleSize.Small]: TitleTag.H3,
-    [TitleSize.ExtraSmall]: TitleTag.H4,
+    [TitleSize.Large]: TitleTag.H2,
```

Comment:
just one question, is removing the `h1` intentional? or maybe can we add `extraLarge` for it? :pray:

Reply:
Usually the websites only have one h1 and the hero component already uses h1, so if it's really necessary, you can just pass the tag to be H1. But if you only pass `size={TitleSize.Large}` then it just fallbacks to `TitleTag.H2` unless tag is specified.

...
This is just to make components easier to use, but I can add the H1 back if you want. Let me know @evavirseda

Reply:
it's ok, just curious :pray: